setup/native-script.rb

```diff
@@ -0,0 +1,63 @@
+# coding: utf-8
+
+# Only the user can manually download and install Xcode from AppStore
+puts "Installing Xcode... Please, click 'Get' or 'Update' to install Xcode from the App Store."
+`open 'macappstore://itunes.apple.com/us/app/xcode/id497799835'`
+
+until `xcodebuild -version`.include? "version" do
+  puts "Waiting for Xcode do finish installing..."
+  sleep(30)
+end
+
+puts "You need to accept the Xcode license agreement to be able to use the compilers. (You might need to provide your password.)"
```

Comment:
Agreed. Make this "You need to accept the Xcode license agreement to be able to use the Xcode command-line tools."